public/js/test/node-unit-tests.js

```diff
@@ -3,6 +3,8 @@
 const glob = require("glob").sync;
 const path = require("path");
 
+require("amd-loader");
```

Comment:
this lets us load the amd rep files. there's magic in there for defining a global define function and doing a naive map to require. it's cool, but i would not trust it unless it was for tests.